### ISSUE TITLE
Difficulty with installing and deploying Nemesis with Vortex

### ISSUE THREAD

**Issue by crichmon** [ShikyoKira/Project-New-Reign---Nemesis-Main#572]
Vortex likes to create empty marker files "__folder_managed_by_vortex" in folders that it creates.  Therefore, when attempting to install and deploy Nemesis from Vortex, Vortex will create those empty marker files throughout the Nemesis folder and subfolders.  This is a problem for Nemesis, because when launching Nemesis, it attempts to process some of these Vortex marker files and throws an error and won't complete behavior generation.  I've worked around this by deleting this Vortex marker files in the Nemesis folders and subfolders, but I believe Vortex developers would say this is not a good idea, and instead Nemesis should be able to filter out certain files in some way when generating behavior.

**Comment by Pickysaurus** [comment]
This is correct. Nemesis should properly filter the directories to exclude any files it does not need to avoid potential issues.
